backend/src/main/java/com/google/sps/servlets/SearchMatchServlet.java

```diff
@@ -142,9 +152,31 @@ private void sendMatchResponse(HttpServletResponse response, Match match) throws
     matchExists.put(JSON_FIRST_PARTICIPANT_USERNAME, match.getFirstParticipantUsername());
     matchExists.put(JSON_SECOND_PARTICIPANT_USERNAME, match.getSecondParticipantUsername());
     matchExists.put(JSON_DURATION, match.getDuration());
-
+    try {
+      emailNotifier = createOrGetEmailNotifier();
+    } catch (GeneralSecurityException e) {
+      e.printStackTrace();
+    }
+    try {
+      emailNotifier.sendMatchEmail(getUsername(), userService.getCurrentUser().getEmail());
+    } catch (MessagingException e) {
+      e.printStackTrace();
+    }
     // Send the JSON back as the response
     response.setContentType("application/json");
     response.getWriter().println(matchExists.toString());
   }
+
+  /** Retrieve user email address via Users API and parse for username */
```

Comment:
include a description of when this method returns null

Reply:
Done